src/main/java/uk/gov/hmcts/darts/cases/mapper/CasesMapper.java

```diff
@@ -94,6 +102,24 @@ public CourtCaseEntity addDefendantProsecutorDefenderJudge(CourtCaseEntity caseE
     @Transactional
     public SingleCase mapToSingleCase(CourtCaseEntity caseEntity) {
         SingleCase singleCase = new SingleCase();
+
+        Optional<CaseRetentionEntity> caseRetentionOptional = caseRetentionRepository
+                                                                    .findTopByCourtCaseAndCurrentStateOrderByCreatedDateTimeDesc(
+                                                                        caseEntity,
+                                                                        String.valueOf(CaseStatus.COMPLETE));
+
+        if (!caseRetentionOptional.isEmpty()) {
+            CaseRetentionEntity caseRetention = caseRetentionOptional.get();
+            RetentionPolicyTypeEntity retentionPolicy = caseRetention.getRetentionPolicyType();
+
+            singleCase.setRetainUntilDateTime(caseRetention.getRetainUntil());
+            singleCase.setRetentionDateTimeApplied(caseRetention.getRetainUntilAppliedOn());
+            singleCase.setRetentionPolicyApplied(retentionPolicy.getPolicyName());
+
+            log.debug("The caseRetention is - CurrentState: {}, CreatedDateTime: {} ", caseRetention.getCurrentState(), caseRetention.getCreatedDateTime());
```

Comment:
When making log statements, try to give them context. If i was looking at a stream of these on the server log, it would be useful to know which case it was referring to.

Reply:
Agreed, however debug log was not required anymore and has been removed